circle.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
 
 test:
   override:
-    - make
+    - make test
```

Comment:
I think this introduces a chicken/egg problem.

To make changes to the parsing service (which uses this repository as a submodule). The blueprint here would need to be merged/updated in master before the parsing service and since the implementation won't be merged or deployed, dredd would fail.